rust/libarkworks/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "libarkworks"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "arkworks"
+path = "src/lib.rs"
+crate-type = ["staticlib", "cdylib"]
+
+[dependencies]
+libc = "0.2"
+rand = "0.8.5"
+ark-bn254 = "0.4.0"
+ark-ff = "0.4.2"
+ark-serialize = "0.4.2"
+ark-ec = "0.4.2"
+
+[profile.release]
```

Comment:
could this be used to reduce binary size?
https://doc.rust-lang.org/cargo/reference/profiles.html#opt-level